<launch>

    <!-- Launch the mapping node -->
    <node pkg="semantic_dsp_map" type="mapping_with_external_data" name="mapping_with_external_data" />

    <!-- Launch the rosbag -->
    <node pkg="rosbag" type="play" name="rosbag" args="$(find semantic_dsp_map)/bag/clip1.bag" output="screen" />

    <!-- Launch the rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find semantic_dsp_map)/rviz/map.rviz" output="screen" />

</launch>